tests/test_service.py

```diff
@@ -131,3 +131,70 @@ def test_get_system_information_without_pkg(self):
         cond = (ReportPortalService.get_system_information('pytest')
                 == expected_result)
         assert cond
+
+    @mock.patch("reportportal_client.service._get_data",
+                mock.Mock(return_value={"id": 123}))
+    def test_start_item(self, rp_service):
+        """Test for validate start_test_item.
+
+        :param: rp_service: fixture of ReportPortal
+        """
+        rp_start = rp_service.start_test_item(name="name",
+                                              start_time=1591032041348,
+                                              item_type='STORY')
+        expected_result = dict(json={'name': 'name',
+                                     'description': None,
+                                     'attributes': None,
+                                     'startTime': 1591032041348,
+                                     'launchUuid': 111,
+                                     'type': 'STORY', 'parameters': None,
+                                     'hasStats': True,
+                                     'codeRef': None},
+                               url='http://endpoint/api/v2/project/item',
+                               verify=True)
+
+        rp_service.session.post.assert_called_with(**expected_result)
+        assert rp_start == 123
+
+    start_item_optional = [
+        ('code_ref', '/path/to/test - test_item', "codeRef",
+         '/path/to/test - test_item'),
+        ('attributes', {'attr1': True}, "attributes",
+         [{'key': 'attr1', 'value': 'True', 'system': False}])
+    ]
+
+    @pytest.mark.parametrize(
+        "field_name,field_value,expected_name,expected_value",
+        start_item_optional)
+    @mock.patch("reportportal_client.service._get_data",
+                mock.Mock(return_value={"id": 123}))
+    def test_start_item_code_optional_params(self, rp_service, field_name,
+                                             field_value, expected_name,
+                                             expected_value):
+        """Test for validate different fields in start_test_item.
+
+        :param: rp_service: fixture of ReportPortal
```

Comment:
Please, align param descriptions to the same level.

Reply:
I tried (see commit history), but pydocstyle breaks build in that case: `D208: Docstring is over-indented`.
BTW do we really need that 79 characters line length limitation, in 2020?

Reply:
You can use line breaks for long descriptions. 79 is good, 120 is used mostly in UI testing.

Reply:
Ok, got what you mean.